Func/Convert.cs

```diff
@@ -10,20 +11,31 @@
 
 namespace ICSharpCode.CodeConverter.Func;
 
-public static class Convert
+public class Convert
 {
+    private readonly ILoggerFactory _loggerFactory;
+
+    public Convert(ILoggerFactory loggerFactory)
+    {
+        _loggerFactory = loggerFactory;
+    }
+
     //
     // Sample data: {"code":"Public Class VisualBasicClass\r\n\r\nEnd Class","requestedConversion":"vbnet2cs"}
     //
     [FunctionName("Convert")]
 #pragma warning disable VSTHRD200 // Use "Async" suffix for async methods - Name must be "Run" for this to work AFAIK
-    public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req, ILogger log)
+    public async Task<IActionResult> Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
+        CancellationToken hostCancellationToken)
 #pragma warning restore VSTHRD200 // Use "Async" suffix for async methods
     {
+        var logger = _loggerFactory.CreateLogger<Convert>();
```

Comment:
Will this violate what the site says about not storimg code in any way?
Exception messages contain fragments of code

Reply:
This is not yet logging anything, this is a verification of ILoggerFactory injection. And yes, that is exactly why I didn't sprinkle any of it in CC itself because we need to have the discussion how to make this logging useful to us without violating that promise (even maybe having two paths - because in the extension it might be useful to surface the information to the user via logging whereas we in the Azure Functions case are only interested in the high-level information)